```
# github.com/pokt-network/pocket-core/app/cmd/cli
app/cmd/cli/query.go:90:7: no new variables on left side of :=
app/cmd/cli/query.go:91:10: no new variables on left side of :=
```

--- a/app/cmd/cli/query.go
+++ b/app/cmd/cli/query.go
@@ -87,8 +87,8 @@ var queryTx = &cobra.Command{
 }
 
 func validatePagePerPageArgs(args []string) (page int, perPage int) {
-	page := 0
-	perPage := 0
+	page = 0
+	perPage = 0
 	if len(args) == 2 {
 		parsedPage, err := strconv.Atoi(args[1])
 		if err != nil {

